lib/Common/ConfigFlagsList.h

```diff
@@ -1492,6 +1492,9 @@ FLAGNR(Boolean, CFG, "Force enable CFG on jshost. version in the jshost's manife
 
 FLAGR(Number, JITServerIdleTimeout, "Idle timeout in seconds to do the cleanup in JIT server", 10)
 FLAGR(Number, JITServerMaxInactivePageAllocatorCount, "Max inactive page allocators to keep before schedule a cleanup", 10)
+
+FLAGNR(Boolean, StrictWriteBarrierCheck, "Check write barrier setting on none write barrier pages", false)
```

Comment:
maybe yes, not sure if there are many difference on the data structures. let me enable it on test build

---
In reply to: [86637769](https://github.com/Microsoft/ChakraCore/pull/1920#discussion_r86637769) [](ancestors = 86637769)